### PR TITLE
Cleaning up CSS layout

### PR DESCRIPTION
**Pull request by pigsflew**
This gets rid of all of the z-index cruft, adds scrollbars to the news
window, themes the scroll bars appropriately, and allows sass to
distinguish when it's been packaged in dev mode so I can blur out
character names and hide profile images for fast screenshotting.

image

To switch modes just run either `yarn run encore dev` or `yarn run encore prod` before you load up electron.

### CODE REVIEW

**Comment by pigsflew** [comment]
addresses [#14](https://github.com/xivapi/ffxiv-launcher/issues/14)

File: public/assets/css/app.css [hunk 1]
```diff
@@ -1,562 +1 @@
-/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (pigsflew) [inline public/assets/css/app.css:1]: This file is generated, you might consider removing it from the git and adding it to gitignore

**Comment by vekien** [comment]
Thanks for the changes! 

I will have a look, I really don’t like the scroll bar on the news being on the right, I think it would be nicer on the left which is why it doesn’t exist at the moment until I looked into it. Another option could be to add a faint back BG to the news so that when it’s open it looks like a panel, rather than a floating scroll bar, it would also make it easier to have different backgrounds without too much worry of test eligibility.

What do you mean by “rid of the z index crufts”? 

I’m fine with generated files in git.

**Comment by pigsflew** [comment]
There were a lot of z-index stuff because almost everything was fixed positioning with z-index manipulation to put it above/below other elements.

The news was therefore scrolling visibly underneath the Nav section.

With this change, nothing has a z-index except modals. The News and Character list are still fixed position, because that's how you get their height/overflow to work properly.

I agree about the scrollbar on the right, I'll see about changing that :)

**Comment by pigsflew** [comment]
That needs testing, I haven't run build yet.

[edit]: Yeah, nope. This makes the lodestone news invisible! I have a feeling it's because rtl inverts the left:0/right:0 thing, but i won't be able to look at this until after work today. Thanks!

**Comment by vekien** [comment]
I've never used `rtl`, annoying it does that.

I wonder if just simply putting a dark bg on the news (eg: `rgba(0,0,0,0.5)`) will make it look enough like a seperate panel. And then in future when there is more "panels" we can look into storing the last opened one (including non open).

**Comment by ArcaneDisgea** [comment]
Apply a dark BG and a drop shadow to make it look like a floating panel I might also suggest having it closed by default if the panel idea is what you aim for.

**Comment by pigsflew** [comment]
I was wrong actually, it does work, just that my work machine is a linux laptop :rofl: I don't want to screw anything up by doing things here, so I want to actually test it when i get home.

**Comment by pigsflew** [comment]
All set, good to go, bar's on the left side now

**Comment by vekien** [comment]
Great work on this, all merged and going to do a release.

Worth noting, its importan to run: `yarn run encore prod` **before** `yarn dist`, otherwise you get the blur'd names in a build. I wonder if we could detect dist somehow? If not, may have to write a little `build.sh` script that combines these, for ease :)

**Pull request merged**